lib/middleware/campaigns/index/response.js

```diff
@@ -36,6 +37,7 @@ module.exports = function sendResponse() {
       return formattedCampaign;
     });
 
-    return res.send({ data });
+    const meta = helpers.util.getMeta(data.length);
```

Comment:
Ah -- this is another outlier per https://github.com/DoSomething/gambit-campaigns/pull/1061#issuecomment-407223840. We don't fetch this list from Contentful, but instead from the cached list of topics, filtering by campaign id -- which is why we're inserting `meta` information here. Supporting this `GET /campaigns` endpoint has been more of convenience for Gambit Admin, but should be ideally be deprecated by passing a `campaignId` parameter to our `topics` endpoint -- or better yet, exposing our Contentful index endpoints to GraphQL to DRY